From 02eb264aa2db20fac479a749c39044cc9ed304b2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pasi=20Pet=C3=A4j=C3=A4j=C3=A4rvi?= <pasi.petajajarvi@qt.io>
Date: Wed, 21 Feb 2018 11:26:09 +0200
Subject: [PATCH] Fix build when ftp feature is disabled

Configuring Qt with -no-feature-ftp cause build to fail.

Change-Id: I47f1cdc400702d0211a9f620c8606983f08fa70c
Reviewed-by: Oswald Buddenhagen <oswald.buddenhagen@qt.io>
---
 src/network/access/qnetworkaccessfilebackend.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/network/access/qnetworkaccessfilebackend.cpp b/src/network/access/qnetworkaccessfilebackend.cpp
index a5e7daff115..d4ca9c22fc2 100644
--- a/src/network/access/qnetworkaccessfilebackend.cpp
+++ b/src/network/access/qnetworkaccessfilebackend.cpp
@@ -46,6 +46,7 @@
 #include "private/qnoncontiguousbytedevice_p.h"
 
 #include <QtCore/QCoreApplication>
+#include <QtCore/QDateTime>
 
 QT_BEGIN_NAMESPACE
 